public/locales/en/translation.json

```diff
@@ -155,7 +155,7 @@
         "einsteinCup2022": {
           "date": "Mai 30.&thinsp;&ndash;&thinsp;June 02.&nbsp;2022",
           "title": "Albert-Einstein-Cup",
-          "text": "The winners of the Albert Einstein Cup in Neubrandenburg are: Joachim Schiller (Germany), Justin Konratt (Germany) and Kantubhukta Lavanya Sai Kumar (India). Justin Konratt also wins the additional discipline of Speed-Programming"
+          "text": "The winners of the Albert Einstein Cup in Neubrandenburg are: Joachim Schiller (Germany), Justin Konratt (Germany) and Kantabhukta Lavanya Sai Kumar (India). Justin Konratt also wins the additional discipline of Speed-Programming"

```

Comment:
The existing spelling with “u” is already the correct one

Reply:
Ugh, sorry! The image I received was very misleading then